lib/team.rb

```diff
@@ -0,0 +1,28 @@
+require 'csv'
+
+class Team
+
+  @@all = nil
```

Comment:
can be consolidated in an inheritance later down the line

Reply:
agreed

Reply:
bingo